src/drivers/gps/gps.cpp

```diff
@@ -987,6 +1004,42 @@ void
 GPS::publish()
 {
 	if (_instance == Instance::Main || _is_gps_main_advertised.load()) {
+
+		if (_mode == GPS_DRIVER_MODE_UBX) {
+			// populate specific ublox model
```

Comment:
Can you move this to https://github.com/PX4/PX4-Autopilot/blob/15c230d18af7bf2de275cfb4adf907a3b4edf942/src/drivers/gps/gps.cpp#L796?

Reply:
Done.